project/core/templates/static/less/thread.less

```diff
@@ -42,6 +42,19 @@ body {
 
 
 #thread {
+    .thread-wiki-holder {
+        display: none;
+    }
+
+    .scrollbar{
+      height: 95vh;
+      overflow-y: scroll;
+    }
+
+    .force-overflow{
+      min-height: 100vh;
+    }
```

Comment:
Let's auto-format this file for consistency.

Reply:
Done ✅